.github/dependabot.yml

```
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - directory: ci/
+    package-ecosystem: pip
+    schedule:
+      interval: weekly
+    ignore:
+      # https://github.com/coiled/benchmarks/issues/1221
+      - dependency-name: pyspark
+        versions: ["1.5.x"]
```

I think this ought to be this, no?
```suggestion
      - dependency-name: pyspark
        versions: ["3.4.x"]
```

---

3.5.X, but yes